### DISCUSSION TITLE
Rowy Pro 12/2022 Pricing Plan(s)

### DISCUSSION THREAD

**Discussion by Olmatro** [AntlerVC/firetable#1073] [Support / Q&A] [answered]
**Rowy Pro 12/2022 Pricing Plan(s) :**

Hi @harinij , @shamsmosowi ,
Sorry for a long off time, my job was needing me far from my code lab, and congratulations for your Rowy Pro 12/2022 Discord announcements !

Where can I find Rowy Pro 12/2022 Pricing Plan(s) ?

Many thanks,

olπ

**Comment by harinij** [comment, marked as answer]
Hi @Olmatro 👋 thank you! you can find Rowy pro info here: https://www.rowy.io/pricing

**Reply by Olmatro** [reply]
Many thanks @harinij !
oLπ
